material/problems/Kinematics/SwingingShip/SwingingShip.tex

```diff
@@ -24,48 +24,110 @@
   Hence, the angular motion of the tire is used to swing the ship back and
   forth.
   %
-  \\[0.5pc]
+  \\[0.45pc]
   During a certain interval, the \textbf{counter-clockwise} rotation rate of
-  the tire is described by $\omega_A = 10*\sin(t)$.
-  The radius of the tire is $R_A = 0.3$ m.
-  %
-  \\[0.5pc]
+  the tire is described by $\omega_A = 10\ast\sin(t)$.
```

Comment:
multiplication should be with `\mult`. Also, with a space between 10, \mult, and \sin